cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -1427,17 +1427,18 @@ GfErrType ThinClientPoolDM::sendSyncRequest(
             GF_SAFE_DELETE_CON(conn);
           }
           excludeServers.insert(ServerLocation(ep->name()));
-          if (error == GF_IOERR) {
+          if (error == GF_IOERR || error == GF_TIMEOUT) {
```

Comment:
Can you combine these nested conditions into one and remove a whole level of indentation?

Reply:
+1! And the same in line 2355